libkineto/src/plugin/xpupti/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# xpupti: XPU implementation for kineto profiler
+# outputs:
+#  SYCL_INCLUDE_DIR   --  SYCL include dir
+#  SYCL_LIBRARY       --  SYCL library file
+#  XPU_xpupti_LIBRARY --  XPUPTI dependencies lib
+#  XPUPTI_BUILD_FLAG  --  XPUPTI build flags
+
+if((NOT SYCL_INCLUDE_DIR) OR (NOT SYCL_LIBRARY_DIR))
+    include(FindSYCLToolkit.cmake)
+    if(NOT SYCLTOOLKIT_FOUND)
+        message(FATAL_ERROR "${SYCL_NOT_FOUND_MESSAGE}")
+    endif()
+endif()
+
+message(INFO " SYCL_INCLUDE_DIR = ${SYCL_INCLUDE_DIR}")
+message(INFO " SYCL_LIBRARY = ${SYCL_LIBRARY}")
+
+set(XPUPTI_INCLUDE_DIR ${SYCL_INCLUDE_DIR} PARENT_SCOPE)
+
+# find xpupti sdk
+find_package(Pti)
```

Comment:
The Pti is a MUST component. Should we add the "required" flag here?

Reply:
ok